### PR TITLE
Fix issues #237 #296

### PR DESCRIPTION
**Pull request by jcrivenaes**
In addition to fixes, also pydocstyle linting.

### CODE REVIEW

**Comment by ertomatic** [comment]
Can one of the admins verify this patch?

File: src/subscript/runrms/runrms.py [hunk 1]
```diff
@@ -199,6 +199,7 @@ class RunRMS:
     # pylint: disable=too-many-instance-attributes
 
     def __init__(self):
+        """Instantate object."""
```
Comment (berland) [inline src/subscript/runrms/runrms.py:202]: Instantiate

File: src/subscript/runrms/runrms.py [hunk 2]
```diff
@@ -339,25 +342,28 @@ def requested_rms_version(self):
...
 
         logger.debug("EXECUTABLE: %s", self.exe)
         logger.debug("PYTHONPATH: %s", self.pythonpath)
+        logger.debug("PLUGINSPATH: %s", self.pluginspath)
         logger.debug("TCLTK: %s", self.tcltkpath)
 
-    def _process_pypath(self, pypath):
-        """
-        The proposed pythonpath or testpythonpath from setup is a list.
+    def _process_genericpath(self, pypath):
```
Comment (berland) [inline src/subscript/runrms/runrms.py:365]: Function name and docstring are not fully compatible, is this a function getting something or a function processing something?

Is it perhaps a valdation function?

**Review by berland — APPROVED** [review]
Ok!

**Pull request merged**